server/backend/api/app/crud/crud_utils.py

```diff
@@ -11,6 +12,13 @@
 
 LOGGER = logging.getLogger(__name__)
 
+def to_upper(elements: List[str]) -> List[str]:
```

Comment:
Probably don't need the to_upper function anymore, but don't let this hold up merging the PR.